src/main/scala/firrtl/passes/LowerTypes.scala

```diff
@@ -75,7 +75,7 @@ object LowerTypes extends Transform with DependencyAPIMigration {
     val memInitByModule = memInitAnnos.map(_.asInstanceOf[MemoryInitAnnotation]).groupBy(_.target.encapsulatingModule)
 
     val c = CircuitTarget(state.circuit.main)
-    val refRenameMap = RenameMap()
+    val refRenameMap = RenameMap.noDistinct() // Hacky, but big performance win
```

Comment:
Can you add a more verbose block comment about this?

Reply:
yep

Reply:
Done.